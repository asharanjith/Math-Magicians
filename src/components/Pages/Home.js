import React from 'react';

const Home = () => {
  <div>
    <h1>Welcome to our page</h1>
    <p>
      Lists of mathematics topics cover a variety of topics related to
      mathematics.
      {' '}
    </p>
  </div>;
};

export default Home;
