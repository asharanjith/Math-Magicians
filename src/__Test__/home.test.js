import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Pages/Home';

describe('check Homepage', () => {
  const tree = renderer.create(<Home />).toJSON();
  test('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
