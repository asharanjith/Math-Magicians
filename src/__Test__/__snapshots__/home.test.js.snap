// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<div
  className="Calculator"
>
  <h2>
    Lets do some Math!
  </h2>
  <div
    className="calculator-display"
  >
    <div
      className="calculator-display__result"
    >
      0
    </div>
    <div
      className="calculator-keypad"
    >
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="calculator-keypad__key operator"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="calculator-keypad__key operator"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="calculator-keypad__key operator"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="calculator-keypad__key operator"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="calculator-keypad__key zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="calculator-keypad__key"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="calculator-keypad__key operator"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;

exports[`check Homepage renders correctly 1`] = `
<div
  className="homePage"
>
  <div
    className="welcome"
  >
    <h1>
      Welcome to our page
    </h1>
    <p>
      Mathematics is the cradle of all creations, without which the world cannot move an inch. Be it a cook or a farmer, a carpenter or a mechanic, a shopkeeper or a doctor, an engineer or a scientist, a musician or a magician, everyone needs mathematics in their day-to-day life. Even insects use mathematics in their everyday life for existence.
    </p>
  </div>
</div>
`;
